main.py

```diff
@@ -1,7 +1,39 @@
+from django.db.models import QuerySet
+
 import init_django_orm  # noqa: F401
+from db.models import Actor, Genre
 
-from django.db.models import QuerySet
+genres = ["Western", "Action", "Dramma"]
+actors = [
+    ("George", "Klooney"),
+    ("Kianu", "Reaves"),
+    ("Scarlett", "Keegan"),
+    ("Will", "Smith"),
+    ("Jaden", "Smith"),
+    ("Scarlett", "Johansson")
+]
 
 
 def main() -> QuerySet:
-    pass
+    for genre in genres:
+        Genre.objects.create(name=genre)
+
+    for first_name, last_name in actors:
+        Actor.objects.create(
+            first_name=first_name,
+            last_name=last_name
+        )
+
+    Genre.objects.filter(name="Dramma").update(name="Drama")
+    Actor.objects.filter(first_name="George").update(last_name="Clooney")
```

Comment:
I meant that you should filter by the first name and the last name, not only the first name